java/impl/gcp/src/main/java/co/worklytics/psoxy/SecretManagerConfigService.java

```diff
@@ -194,6 +182,48 @@ public void release(String lockId) {
         }
     }
 
+    private static void updateLabelFromSecret(SecretManagerServiceClient client, SecretName secretName, String label, String labelValue) {
+        try {
+            client.updateSecret(UpdateSecretRequest.newBuilder()
+                    .setSecret(Secret.newBuilder()
+                            .setName(secretName.toString())
+                            // Label format is https://cloud.google.com/compute/docs/labeling-resources#requirements
+                            .putLabels(label, labelValue)
+                            .build())
+                    .setUpdateMask(FieldMask.newBuilder()
+                            .addPaths("labels")
+                            .build())
+                    .build());
+        } catch (Exception e) {
+            log.severe(String.format("Cannot put the label of the version on the secret %s due exception: %s", secretName.toString(), e));
```

Comment:
don't serialize exception like this; leverage `log.log(Level.SEVERE, ..., e)` to get it via the native support for that.

Reply:
also, imho you shouldn't catch this here, or should at least re-throw.  you're supressing it, so caller won't detect failure and will just continue. That's probably OK in the version labeling case, but not in the lock case - it will believe lock was acquired, right?